tensorflow_serving/tools/docker/Dockerfile.devel

```diff
@@ -28,7 +28,7 @@ RUN curl -fSsL -O https://bootstrap.pypa.io/get-pip.py && \
 
 # Set up grpc
 
-RUN pip install enum34 futures six && \
+RUN pip install enum34 futures mock six && \
     pip install --pre protobuf>=3.0.0a3 && \
```

Comment:
Ok, I confirmed it builds well, the file `=3.0.0a3` is not there anymore and all tests pass.
`Executed 46 out of 46 tests: 46 tests pass.`